packages/http-smoke-testing/src/RouteInfo.php

```diff
@@ -79,4 +86,12 @@ public function getRouteParameterNames()
 
         return $compiledRoute->getVariables();
     }
+
+    /**
+     * @return array|null
+     */
+    public function getAnnotations(): ?array
```

Comment:
This cannot return null.

```suggestion
    public function getAnnotations(): array
```